uitestcore/utilities/browser_handler.py

```diff
@@ -63,17 +64,19 @@ def take_screenshot(driver, description):
 
         # Create a file name and ensure it is not too long
         timestamp = get_current_datetime().strftime("%Y-%m-%d_%H.%M.%S.%f")
+        description = remove_invalid_characters(description)
```

Comment:
I see you've added a test to test_string_util, can you also add one to test_browser_handler where the filename contains a / please?